src/index.scss

```diff
@@ -1,6 +1,7 @@
+@use "@openedx/paragon/styles/css/core/custom-media-breakpoints.css" as paragonCustomMediaBreakpoints;
```

Comment:
[question]: Since we have several options for connecting CSS variables to MFE and some features related to creating/working with design tokens, we will probably need good documentation, which makes sense to write. What do you think?

Reply:
You are right. We need to create a proper documentation in the future.